### DISCUSSION TITLE
明细表使用SeriesNumberCell无法支持多选

### DISCUSSION THREAD

**Discussion by wzdong26** [antvis/S2#2909] [General]
明细表使用在options中加了：seriesNumber: { enable: true }, interaction: { multiSelection: true },
multiSelection不生效，无法使用(Command/Ctrl) + click多选行。这里想实现多选行有什么解决方法吗？

https://github.com/user-attachments/assets/d72c3793-6048-4e1f-9f96-a179b4acad71

**Comment by lijinke666** [comment]
目前不行, 没有实现该能力, 感兴趣的话可以来个 PR.
